src/lib/seam/components/SupportedDeviceTable/SupportedDeviceTable.tsx

```diff
@@ -8,11 +8,13 @@ import type { DeviceModelFilters } from 'lib/seam/components/SupportedDeviceTabl
 export interface SupportedDeviceTableProps {
   cannotFilter?: boolean
   className?: string
+  brands?: string[]
```

Comment:
This needs to be added here as well: https://github.com/seamapi/react/blob/main/src/lib/seam/components/SupportedDeviceTable/SupportedDeviceTable.element.ts#L7